server/service/src/sync/program_schemas/patient_ui_schema.json

```diff
@@ -282,11 +282,13 @@
                           "elements": [
                             {
                               "type": "Control",
-                              "scope": "/properties/address1"
+                              "scope": "/properties/address1",
+                              "options": { "rows": 3 }
                             },
                             {
                               "type": "Control",
-                              "scope": "/properties/address2"
+                              "scope": "/properties/address2",
+                              "options": { "rows": 3 }
```

Comment:
Just so we can see what a multi-row text field looks like. Address seems like an appropriate candidate.